src/main/java/com/github/tomakehurst/wiremock/servlet/WireMockHttpServletRequestAdapter.java

```diff
@@ -289,7 +289,7 @@ public Collection<Part> getParts() {
   @Override
   public boolean isMultipart() {
     String header = getHeader("Content-Type");
-    return (header != null && header.contains("multipart/"));
+    return (header != null && header.matches("(?i)^multipart/.*"));
```

Comment:
I agree. I've updated the regular Expression to ignore any whitespace character at the beginning of content-type

Reply:
Nice, one. Can you keep both, with and without spaces?😃

Reply:
This regular Expression already handles both cases

Reply:
Or did you mean the Test Case?

Reply:
yeah, I meant the test case,hehe

Reply:
Ok, i've adapted the Test so that the same Content will be sent twice. One time with leading spaces before the content-type and one time without leading spaces